flutter/lib/resources/result_manager.dart

```diff
@@ -3,58 +3,31 @@ import 'dart:io';
 
 import 'package:mlperfbench/benchmark/benchmark.dart';
 import 'package:mlperfbench/benchmark/benchmark_result.dart';
-import 'package:mlperfbench/protos/backend_setting.pb.dart' as pb;
 
 class _BriefResult {
-  static const _idTag = 'benchmark_id';
-  static const _scoreTag = 'score';
-  static const _accuracyTag = 'accuracy';
-  static const _shardsNumTag = 'shards_num';
-  static const _batchSizeTag = 'batch_size';
-  static const _backendDescriptionTag = 'backend_description';
+  static const _tagId = 'benchmark_id';
+  static const _tagPerformance = 'performance';
+  static const _tagAccuracy = 'accuracy';
 
   final String id;
-  final double? score;
-  final String? accuracy;
-  final int shardsNum;
-  final int batchSize;
-  final String backendDescription;
+  final BenchmarkResult? performance;
+  final BenchmarkResult? accuracy;
```

Comment:
The name `performance` and `accuracy` are cryptic when accessed outside this class. Please rename to `performanceModeResult` and `accuracyModeResult`.

Reply:
Done